includes/3rdparty/elementor/class-elementor.php

```diff
@@ -98,7 +98,7 @@ public function start_capturing() {
 	 */
 	public function end_capturing() {
 		$output  = \ob_get_clean();
-		$search  = '/(<div class="elementor-component-tab elementor-panel-navigation-tab" data-tab="global">.*<\/div>)/m';
+		$search  = '/(<(div|button) class="elementor-component-tab elementor-panel-navigation-tab" data-tab="global">.*<\/(div|button)>)/m';
 		$replace = '${1}<div class="elementor-component-tab elementor-panel-navigation-tab" data-tab="rank-math">SEO</div>';
```

Comment:
The code:
```php
$search  = '/(<(div|button) class="elementor-component-tab elementor-panel-navigation-tab" data-tab="global">.*<\/(div|button)>)/m';
```

Works.

But, we need to make the changes in line:
```php
$replace = '${1}<div class="elementor-component-tab elementor-panel-navigation-tab" data-tab="rank-math">SEO</div>';
```

See now, with the new plugin it adds the `<button>` tag for Elementor buttons and the `<div>` tag for Rank Math:
https://i.rankmath.com/i/fXSmxX

The buttons need to same as Elementor buttons.